cmd/soroban-cli/src/commands/keys/generate.rs

```diff
@@ -1,3 +1,5 @@
+use crate::commands::global;
+use crate::print::Print;
 use clap::{arg, command};
 
```

Comment:
We try to keep external deps at the top.

```suggestion
use clap::{arg, command};

use crate::{commands::global, print::Print};
```